src/ripple/overlay/impl/PeerImp.h

```diff
@@ -127,8 +128,8 @@ class PeerImp
     LedgerIndex maxLedger_ = 0;
     uint256 closedLedgerHash_;
     uint256 previousLedgerHash_;
-    std::list<uint256> recentLedgers_;
-    std::list<uint256> recentTxSets_;
+    std::deque<uint256> recentLedgers_;
```

Comment:
Perhaps a note to explain why we're using deque here rather than vector (least surprise?)